test.py

```diff
@@ -2,13 +2,25 @@
 """
 Some basic tests to verify that the wrapper is working
 """
+from os import getenv
+
+from dotenv import load_dotenv, find_dotenv
```

Comment:
I'm happy to add the documentation for supplying credentials via a .env file, but I'm a little confused. The test.py file does not currently run without modification. It attempts to build a client using string literals "MAILCHIMP_USER" and "MAILCHIMP_SECRET", which are invalid (at the very least, there's no data center string there so the requests are made to mailchimp_secret.api.mailchimp.com, and fail since that doesn't resolve). Am I missing something?